AdminCore.WebApp/Controllers/ProjectController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using AdminCore.Common.Interfaces;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using AdminCore.DTOs.Project;
+using AdminCore.WebApi.Models.Client;
+using AdminCore.WebApi.Models.Project;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+
+namespace AdminCore.WebApi.Controllers
+{
+  [ApiController]
+  [Route("[controller]")]
+  public class ProjectController : BaseController
+  {
+    private readonly IProjectService _projectService;
+
+    public ProjectController(IProjectService projectService, IMapper mapper) : base(mapper)
+    {
+      _projectService = projectService;
+    }
+
+    private IActionResult HandleBadRequest(Func<IActionResult> serviceAction)
```

Comment:
private function should probably not live between public functions, more suitable place would be bottom of the class.